reward/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "rad-reward"
+version = "0.1.0"
+authors = ["The Radicle Team <dev@radicle.xyz>"]
+edition = "2018"
+license = "MIT OR Apache-2.0"
```

Comment:
Everything that depends on librad is defacto GPLv3. I fixed this in the rest of the repo recently: https://github.com/radicle-dev/radicle-cli/commit/c1538a3ae8ad2cf7e34aa42c879ca235c37fda00

Reply:
https://github.com/radicle-dev/radicle-cli/compare/30805149493b3e5bd4eebcd5afb5746c9b987c3f..e9e132c44330b15ae72aa3ea6dbbae51b7d2058f#diff-800cd58e22704bf1eff461125a2148f684ee5bebc98d59411501468f7e99df63R6